modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
Thank you Alex. So would that be safe to assume that OPENCV_DNN_TEST_DATA_PATH is set on the auto-build environments? May I rely on that?

Lat question - is it possible to move space_shuttle to testdata/dnn? I mean since there is a general rule to store test data there then I guess no point to have any test images in the main repository?